src/onnx/parse_convolution.cpp

```diff
@@ -149,9 +168,25 @@ struct parse_convolution : op_parser<parse_convolution>
 
         recalc_conv_attributes(values, kdims);
 
+        instruction_ref ret;
+        // parse a_zero_point and b_zero_point values
+        auto is_quant_conv = opd.op_name == "quant_convolution";
+        auto x_zp          = set_bias(x, info, 2, is_quant_conv, args);
+        auto w_zp          = set_bias(weights, info, 3, is_quant_conv, args);
+
         op.from_value(values);
-        auto l1 = info.add_instruction(op, l0, args[1]);
-        return info.add_bias(args, l1, 1);
+
+        x_zp->debug_print();
+
+        ret = info.add_instruction(op, x_zp, w_zp);
```

Comment:
I am not sure we should be passing in x and zp thats been offset by the zero points here, in most cases this will cause the original weight and input to overflow. 
The proper way to handle zero points would be to do the computation using the original input/weight and then computing the offset to be applied to the result and then apply that offset.

Reply:
Conv(x,w) - Conv(x_zp, w_zp) is what you mean? Is the goal then to hope these get fused?

Reply:
@shivadbhavsar added this and updated parser coverage to handle.